build_library/qemu_template.sh

```diff
@@ -198,7 +198,8 @@ if [ -n "${VM_UUID}" ]; then
 fi
 
 if [ -n "${VM_CDROM}" ]; then
-    set -- -cdrom "${SCRIPT_DIR}/${VM_CDROM}" "$@"
+    set -- -boot once=d \
```

Comment:
Looking at this more closely, this should be `order=d` rather than `once=d`.  Otherwise rebooting the machine will fail if a hard disk is specified on the command line.

Reply:
@bgilbert Done. I changed it to `order=d`.